lib/v4/errorMap/invalidValue.ts

```diff
@@ -11,11 +12,16 @@ export function parseInvalidValueIssue(
     | 'maxAllowedValuesToDisplay'
     | 'wrapAllowedValuesInQuote'
     | 'allowedValuesLastSeparator'
+    | 'reportInput'
+    | 'numberLocalization'
+    | 'dateLocalization'
   >
 ): AbstractSyntaxTree {
   let message: string;
 
-  if (issue.values.length === 0) {
+  if (issue.expected === 'stringbool') {
```

Comment:
question: how come we added this here?

Reply:
I wanted to render a different message with `stringbool` failures, e.g. `expected boolean as string, received "foobar"`. Not sure if this answers the question.